p2p/raintree/network_test.go

```diff
@@ -0,0 +1,67 @@
+package raintree
+
+import (
+	"testing"
+
+	"github.com/pokt-network/pocket/p2p/types"
+	typesP2P "github.com/pokt-network/pocket/p2p/types"
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_rainTreeNetwork_AddPeerToAddrBook(t *testing.T) {
```

Comment:
Addressed in https://github.com/pokt-network/pocket/pull/222/commits/ccc4e81b25343d163c2052fcb834620f654e3f7d

Reply:
PS:
I didn't rename the tests `TestRainTreeAddrBookTargets...`  because I wanted to have something to compare like for like with what was in main, anyway, once you are happy with all the changes and this is ready to merge, we can rename these tests once and for all.